values.yaml

```diff
@@ -442,6 +442,12 @@ server:
     #    hosts:
     #      - chart-example.local
 
+  # hostAliases is a list of aliases to be added to /etc/hosts. Specified as a YAML list.
+  hostAliases: null
```

Comment:
I think we could default this to an empty list, and that would make the json schema definition a little simpler (no `null` needed).

```suggestion
  hostAliases: []
```

Reply:
@tvoran thanks for taking a look! Sorry for the delay, things have been a bit busy for me but I'll add some tests and update the PR with the schema suggestion soon.